crates/kitsune_p2p/dht/tests/arc_stability.rs

```diff
@@ -63,7 +62,6 @@ fn stability_test_case_near_ideal() {
 
 #[test]
 fn stability_test_case_messy() {
-    std::env::set_var("RUST_LOG", "debug");
```

Comment:
Was there a reason for this? I had issues because of #3651 which prevented the tests running together with this here. I could only run them one at a time and couldn't see the panics because this outputs so much information.